torch/_inductor/codecache.py

```diff
@@ -713,8 +713,8 @@ class BypassFxGraphCache(Exception):
     Exception to indicate that the FxGraphCache should be bypassed.
     """
 
-    pass
-
+    def __init__(self, message: str = ""):
+        super().__init__(message)
```

Comment:
since this inherits Exception, you dont need this init method

Reply:
Done.